### ISSUE TITLE
criar carrossel de cursos na página javascript

### ISSUE THREAD

**Issue by felipepassosc** [felipepassosc/devhipster#8]

**Comment by leogaletti** [comment]
Esse Carrossel seria ao lado do vídeo? Tem uma ideia de como seria?

**Comment by felipepassosdev** [comment]
https://skylab.rocketseat.com.br/node/curso-java-script esse é um ótimo exemplo, estava pensando em algo parecido

**Comment by leogaletti** [comment]
Se puder mandar um print, acho que essa área é privada pra quem tem acesso ao Skylab da Rocketseat

**Comment by leogaletti** [comment]
Dai já pode associar essa tarefa a mim que já crio o Carrossel na página. Vou criar como componente pra ser utilizado em outras telas também, o que acha? Ou vai ser algo específico dessa tela?

**Comment by felipepassosdev** [comment]
![image](https://user-images.githubusercontent.com/38237206/82245439-548d4d00-9919-11ea-8a46-b7a2343bacc6.png)

**Comment by felipepassosdev** [comment]
Perfeito, a ideia é essa mesmo, vou até muda o título

**Comment by felipepassosdev** [comment]
Posso deixar essa tarefa pra ti? lembrando do uso da biblioteca que decidimos

**Comment by leogaletti** [comment]
Claro! Vou trabalhar hoje a noite nisso, dai já uso a biblioteca e a nova estrutura de arquivos

**Comment by leogaletti** [comment]
Componente criado com uma listagem básica. Vou agora customizar o visual e colocar os checks da aula já visualizada.

**Comment by felipepassosdev** [comment]
valeuuuu irmão, manda balaaaaaaaa

**Comment by felipepassosdev** [comment]
Eae mano,como está o desenvolvimento dessa aba, precisa de alguma ajuda? só chamar meu mano @leogaletti

**Comment by leogaletti** [comment]
Cara só não terminei hoje porque foi pura correria, mas amanhã a tarde e a noite com mais calma finalizo e mando pra cá e já meto bala no backend!

**Comment by felipepassosdev** [comment]
se alguém tiver interesse, se não eu começo o desenvolvimento
